pkg/metrics/metrics.go

```diff
@@ -67,6 +69,7 @@ func initMetrics() {
 	// Gauges with labels
 	UserAccountGaugeVec = newGaugeVec("user_accounts_current", "Current number of User Accounts (per member cluster)", "cluster_name")
 	UsersPerActivationGaugeVec = newGaugeVec("users_per_activations", "Number of users per activation", []string{"activations"}...)
+	MasterUserRecordGaugeVec = newGaugeVec("master_user_records", "Number of Master User Records (per domain)", "domain")
```

Comment:
I wonder if we should use `domain` or `email_domain`. The latter is more specific, so it would avoid conflict/confusion if we have another kind of "domain" in the future. WDYT?

Reply:
I think both email_domain or domain would work. I'm OK to keep it shorter as "domain" but feel free to rename it if you like email_domain better.

Reply:
Ok, thanks for your feedback. I've kept the label short (`domain`), but I clarified its purpose in the metric description in c0d0906